src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/UpdateDto.java

```diff
@@ -29,6 +30,9 @@ public class UpdateDto {
     public static final String TRUST_DATA_FETCHED = "trust_data_fetched";
 
     public static final String REVIEW_TRUSTS_FIELD = "review_trusts";
+    private static final String RELEVANT_STATEMENT1 = "relevant_period_beneficial_owner";
+    private static final String RELEVANT_STATEMENT2 = "relevant_period_trust_involved";
+    private static final String RELEVANT_STATEMENT3 = "relevant_period_trust_beneficiary";
```

Comment:
Should these be the new names `ceased_to_be_registrable_beneficial_owner`, `trust_involved_in_the_oe`, `become_or_ceased_beneficiary_of_a_trust`

Reply:
Done, thanks